### PR TITLE
Handle refactor

### PR DESCRIPTION
**Pull request by thehowl**
As per discussion in #3330

### CODE REVIEW

File: routers/routes/routes.go [hunk 1]
```diff
@@ -353,10 +353,6 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Group("", func() {
 			m.Get("/create", org.Create)
 			m.Post("/create", bindIgnErr(auth.CreateOrgForm{}), org.CreatePost)
-		}, func(ctx *context.Context) {
-			if !ctx.User.CanCreateOrganization() {
-				ctx.NotFound()
-			}
```
Comment (thehowl) [inline routers/routes/routes.go:359]: Before anyone asks why this was removed: ctx.NotFound does not do what you expect it to - it does not return a 404 response. It registers the handlers for 404 responses, and in this case resets them. The logic for returning not found when someone tries to access /org/create has been moved to org.Create and org.CreatePost.

**Review by lafriks — APPROVED** [review]

**Review by appleboy — APPROVED** [review]

**Review by sapk — APPROVED** [review]
CI fail seems unrelated

**Comment by lafriks** [comment]
~~@thehowl can you please force push?~~ nevermind, I can now restart drone builds :)

**Comment by lunny** [comment]
@thehowl sorry for late to submit my review.

**Comment by thehowl** [comment]
@lunny thanks for the review, will address issues later today...

**Pull request merged**